packages/php-wasm/compile/php/phpwasm-emscripten-library.js

```diff
@@ -264,15 +268,29 @@ const LibraryExample = {
 		if (!command) {
 			return 1;
 		}
-
 		const cmdstr = UTF8ToString(command);
 		if (!cmdstr.length) {
 			return 0;
 		}
 
+		let jsArgsArray = [];
+		if (argsLength) {
+			var ptr = args + (((argsLength > 1 ? argsLength : 2) >> 1) + 1) * 8;
+			for (var i = 0; i < argsLength; i++) {
+				var str = UTF8ToString(ptr);
+				ptr +=
```

Comment:
Wild! What are these calculations? A comment would be really helpful, especially if it contained a link to some manual page somewhere where this is explained.

Reply:
You're right, wild is the term. I will explain that part in the next commit, this was not easy to implement.